src/supernode/src/main/java/com/dragonflyoss/dragonfly/supernode/common/util/HttpClientUtil.java

```diff
@@ -84,8 +82,9 @@ public static boolean isExpired(String fileUrl, long lastModified, String eTag,
                 conn = openConnection(url);
                 fillHeaders(conn, headers);
                 conn.setUseCaches(false);
+                //conn.setRequestMethod("HEAD");
                 conn.setConnectTimeout(2000);
-                conn.setReadTimeout(1000);
+                conn.setReadTimeout(2000);
```

Comment:
How did you get this value? I think it is unreasonable if it is based on experience. 
Could we make it configurable? 
+ If that, we have to do that without hesitation . 
+ If not, let's take a step back and set a constant.

WDYT?

Reply:
Keep the same with the value in function `getContentLength`. I have no plan to make it configurable in java-version. Maybe we could do that when we refactor supernode with golang.

Reply:
Then set a constant is better. WDYT?

Reply:
I think that this work can be done in supernode go-version, and java-version we just do some bugfix work.

Reply:
> I think that this work can be done in supernode go-version, and java-version we just do some bugfix work.

agree